src/main/scala/io/github/gaelrenoux/tranzactio/doobie/package.scala

```diff
@@ -47,17 +44,20 @@ package object doobie extends Wrapper {
     private[tranzactio] override implicit val connectionTag: Tag[Connection] = doobie.connectionTag
 
     /** How to provide a Connection for the module, given a JDBC connection and some environment. */
-    final def connectionFromJdbc(env: Blocking, connection: JdbcConnection): ZIO[Any, Nothing, Connection] =
-      ZCatsBlocker.provide(env).map { b =>
-        val connect = (c: JdbcConnection) => Resource.pure[Task, JdbcConnection](c)
-        val interp = KleisliInterpreter[Task](b).ConnectionInterpreter
-        val tran = Transactor(connection, connect, interp, Strategy.void)
-        Has(tran)
-      }
+    final def connectionFromJdbc(env: TranzactioEnv, connection: JdbcConnection): ZIO[Any, Nothing, Connection] = {
+      ZIO.runtime[TranzactioEnv].flatMap { implicit r: zio.Runtime[TranzactioEnv] =>
+        ZIO.succeed[Connection] {
```

Comment:
@hmemcpy I'm using `succeed` instead of `effectTotal` here, because I don't think there's any effecting code going under the hood here. Am I missing something ?